policy-test/tests/outbound_api.rs

```diff
@@ -623,6 +647,251 @@ async fn opaque_service() {
     .await;
 }
 
+#[tokio::test(flavor = "current_thread")]
+async fn route_with_filters() {
```

Comment:
nit: it might be more accurate to call this test "*rule* with filters", rather than "route with filters", since the filters are part of the route in both this case and the backend case. :woman_shrugging: